pkg/operator/ceph/cluster/cluster.go

```diff
@@ -397,6 +411,66 @@ func (c *ClusterController) preClusterStartValidation(cluster *cluster, clusterO
 	return nil
 }
 
+func (c *cluster) removeDefaultReplicationRule() error {
+	err := c.context.Executor.ExecuteCommand("ceph", "osd", "crush", "rule", "rm", "replicated_rule")
+	if err != nil {
+		if exitErr, ok := err.(*exec.ExitError); ok && exitErr.ExitCode() == int(syscall.EBUSY) {
+			logger.Debug("replicated_rule is in use, not removed")
+			// ignore this error condition
+			err = nil
```

Comment:
A more common pattern, at least in rook, is to just return immediately.
```suggestion
			return nil
```